### PR TITLE
Provide CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by xirc**
It would be better to provide `CONTRIBUTING.md`.
It contains great materials for us.

See also: [Welcome Contributors](https://docs.scala-lang.org/overviews/contributors/index.html#welcome-contributors)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+sbt test
+```
+
+## Run Integration Tests
+```shell
+sbt multi-jvm:test
+```
+
+If we want to dig in failed test cases in integration tests,
+we can look at integration test reports `target/multi-jvm-test-reports/*.xml` using [xunit-viewer](https://www.npmjs.com/package/xunit-viewer).
```
Comment (xirc) [inline CONTRIBUTING.md:29]: I picked it up from
> Enable JUnit XML Reporter in MultiJVM https://github.com/lerna-stack/akka-entity-replication/pull/28

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Tips: Integration tests is not stable for now.
+
+## Take test coverage
+```shell
+sbt testCoverage
+```
+
+A test coverage is generated in the directory `target/scala-2.13/scoverage-report`.
```
Comment (xirc) [inline CONTRIBUTING.md:38]: I picked it up from
> Check test coverage https://github.com/lerna-stack/akka-entity-replication/pull/25

**Comment by negokaz** [comment]
@xirc 

Great! 👍 
How about linking to this from the README.md?

**Comment by xirc** [comment]
@negokaz 
> How about linking to this from the README.md?

Great idea.

I have committed.
> Link CONTRIBUTING from README 2fd9cd9

**Review by negokaz — APPROVED** [review]

**Pull request merged**